app-android/features/wardrobe/src/main/kotlin/com/closet/features/wardrobe/ClothingDetailScreen.kt

```
@@ -271,32 +365,284 @@ private fun ClothingDetailContent(
 }
 
+@Composable
+private fun ColorChip(color: ColorEntity) {
+    AssistChip(
+        onClick = {},
+        label = { Text(color.name) },
+        leadingIcon = {
+            val hex = color.hex
+            val chipColor = try {
+                if (hex != null) Color(hex.toColorInt()) else Color.Transparent
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)